kvm/requests.yaml

```diff
@@ -4,7 +4,7 @@ releases:
   - name: chart-operator
     version: ">= 2.20.1"
   - name: kube-state-metrics
-    version: ">= 1.9.0"
+    version: ">= 1.11.0"
```

Comment:
probably same as azure for kvm

Reply:
but what's the block condition then? we already have `> 16.2.0`

Reply:
yeah correct, this one is ok